File: src/models/player.ts
export default class Player {
  name!: string;
  team!: string;
  position!: string;
  gamesPlayed!: number;
  gamesStarted!: number;

  constructor(json: any) {
    this.name = json.player;
    this.team = json.team;
    this.position = json.pos;
    this.gamesPlayed = json.g;
    this.gamesStarted = json.gs;
  }
}

File: c-sharp/src/models/Player.cs
namespace MaddenImporter
{
	public abstract class Player
	{
		public string Name { get; set; }
		public string Team { get; set; }
		public string Position { get; set; }
		public int GamesPlayed { get; set; }
		public int GamesStarted { get; set; }
	}
}
